grails-app/domain/gr8di/backend/Message.groovy

```diff
@@ -9,11 +9,14 @@ class Message {
     String topic
     String messageBody
     Date dateSent = new Date()
+    String status
 
     static constraints = {
         senderEmail blank:false
         topic blank:false
         messageBody blank:false
         dateSent blank:false
+        status blank: false, inList: ["read", "unread", "deleted"]
```

Comment:
Would it make sense to turn this into an enum?

Reply:
oh yes, working on that